src/backend/crew_ai/cleaned_llm_wrapper.py

```
@@ -238,61 +132,62 @@ def _clean_llm_result(self, result: LLMResult) -> LLMResult:
     
     Args:
         model_provider: "local" for Ollama, "online" for Gemini
         model_name: Model identifier (e.g., "llama3.1", "gemini-2.5-flash")
         api_key: API key for online models (optional, can use env var)
         
     Returns:
-        Cleaned LLM wrapper instance
+        Cleaned LLM wrapper instance ready for use with CrewAI
     """
     if model_provider == "local":
         logger.info(f"🧹 Creating CleanedOllamaLLMWrapper for model: {model_name}")
```

Ruff (0.14.14)

Logging statement uses f-string

(G004)